converters/toZigbee.js

```diff
@@ -3862,6 +3862,20 @@ const converters = {
             await entity.read('manuSpecificTuya_3', ['switchType']);
         },
     },
+    tuya_power_on_behavior: {
+        key: ['tuya_power_on_behavior'],
```

Comment:
Please rename to `power_on_behavior`

Reply:
There is also `moes_power_on_behavior` converter used with this device which already uses `power_on_behavior` key. So if I rename it the `moes_power_on_behavior` converter will intercept `tuya_power_on_behavior` converter. Or do you meanconverter name, not a key?